chipper/pkg/wirepod/ttr/kgsim_cmds.go

```diff
@@ -476,7 +476,11 @@ func DoGetImage(msgs []openai.ChatCompletionMessage, param string, robot *vector
 		c = openai.NewClientWithConfig(conf)
 	} else if vars.APIConfig.Knowledge.Provider == "openai" {
 		c = openai.NewClient(vars.APIConfig.Knowledge.Key)
-	}
+	} else {
```

Comment:
Can you add the condition
```
else if vars.APIConfig.Knowledge.Provider == "custom"
```
Just to keep it consistent and if other nuanced providers get added it wont fall into their use case

Reply:
Thanks. Fixed in latest draft.